HARK/ConsumptionSaving/ConsRiskyAssetModel.py

```diff
@@ -61,6 +61,11 @@ def __init__(self, verbose=False, quiet=False, **kwds):
         if not hasattr(self, "PortfolioBisect"):
             self.PortfolioBisect = False
 
+        # Boolean determines whether, when simulating a given time period,
+        # all agents will draw the same risky return factor (true by default)
+        if not hasattr(self, "sim_common_Rriksy"):
```

Comment:
I like `Rrisky` because `R` is the return factor and risky is its characteristic. `Rfree` is `Rfree` and not `Free`. Is this a battle for another hill?

Reply:
No, I mean it's misspelled. You spelled R-RIKsy.

Reply:
Crap. You are right. Thanks!